src/Illuminate/Database/Query/Builder.php

```diff
@@ -1759,6 +1759,37 @@ public function orWhereJsonLength($column, $operator, $value = null)
         return $this->whereJsonLength($column, $operator, $value, 'or');
     }
 
+    /**
+     * Apply the where condition if the given $condition is truthy.
+     * If the rest of the $arguments are closures then whereIf() exactly act like when() method.
+     *
+     * @example assume $check = true then
```

Comment:
Please remove this from the code, and put this in a PR to the docs repo. :)

Reply:
Done.